tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Explore SAP Graph on the SAP Graph Navigator)]
+
+The SAP Graph API is documented in the new [SAP Graph Navigator](https://navigator.graph.sap/entities). Go ahead and follow the link to the SAP Graph Navigator application to explore the API. The index, on the left of the application, shows you over 1,500 entities grouped by data source. The first group of entities are the unified entities. Here is an example of a unified entity, `sap.graph/WorkAssignment`.
+
+!![WorkAssignment unified entity in SAP Graph Navigator](blog1_navigator_sm.png)
+
+In this tutorial, you will create a small application that fetches the unified entity `SalesQuote`. Use the SAP Graph Navigator to look up the definition, and corresponding attributes for `SalesQuote`.
+
+[DONE]
+
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Prepare the Node Development Environment)]
+
+We will use **Node.js** and **npm** to build this application, and to keep it extremely simple, we will use the **Express** web framework. Not familiar with Node.js and Express? We recommend this article: <https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction>. That article also explains how to easily set up a Node development environment, whether you use Windows, MacOS or Linux.
+
+If you prefer to develop in the cloud without installing anything on your computer, SAP offers a great cloud-based development experience alternative in the form of the SAP Business Application Studio. Follow [this tutorial](appstudio-onboarding) for more details on getting your own instance. The rest of this tutorial assumes that you installed Node.js and npm on your computer.
+
+[DONE]
+
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Prepare package.json)]
+
+Select a folder (directory) where you want to develop your application. In this folder, you will create a small file called `package.json`, with the following content:
+
+```
+{
+    "name": "hello-graph",
+    "version": "0.0.0",
+    "private": true,
+    "dependencies": {
+        "express": "4.17.1",
+        "node-fetch": "2.6.1",
+        "universal-cookie": "4.0.4"
+    }
+}
+```
+After saving the file, run the following command on your console:
+
+```
+ npm install
+```
+If your node.js environment was properly set up, then this command will install a few standard library packages in a new sub-folder called `node_modules`.
+
+Now, create another sub-folder, next to `node_modules`, called `src`. This is where we will develop our code.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Prepare graph.js)]
+
+First, we will create a boilerplate file in the `src` folder, called `graph.js`. This file will contain a class called `Graph`, which provides a nice wrapper for using SAP Graph. At this time, we will show how to read data, via a `get()` function.
```

Comment:
Use You and not the royal We.(unless, in rare cases, you are referring to SAP).

Reply:
Rephrased.